### ISSUE TITLE
Sorting option does nothing

### ISSUE THREAD

**Issue by Tywele** [jocmp/capyreader#536]
### Steps to Reproduce

1. Change sorting option to "oldest first"
2. Refresh
3. Articles are still sorted by newest first

### Expected behavior

I expect the articles to be sorted by oldest first after changing the option

### Device, Operating system, and app version

Pixel 7
Android 15
App Version 2024.11.1073

**Comment by jocmp** [comment]
hey @Tywele. This feature only sorts oldest first on the "Unread" filter. Are you seeing issues when sorting there?

**Comment by Tywele** [comment]
What's the reason for only sorting the unread view? That doesn't make any sense.

**Comment by Tywele** [comment]
Well then I propose to add sorting for the other feeds as well.

**Comment by jocmp** [comment]
that's fair. I'm going to keep it as-is right now given other open tasks. I may revisit this in the future.

**Comment by mikelococo** [comment]
FWIW, I also found this behavior quite confusing when I encountered it. I realize the option is called `Unread Sort` but my mental model for a global setting is so strongly inclined toward the belief that it would apply to all views that I was super confused for quite some time about what was going on until I found this report.

In addition to the UX confusion, there's also a practical issue of casual debuggability that is complicated by inconsistent behavior across feed views. When I encounter something unexpected in a feed view (and I've encountered a lot of such things in my first few hours experimenting with capyreader/freshrss-greader) my typical approach is to start toggling view features to see how they interact. If each button (or whatever) impacts exactly one feature I can often differentially diagnose a bug (or just my own confusion) in a specific feature or specific interaction between two features. When I try that approach with an oldest-first unread view in capyreader... the sort order toggles every time I switch read/unread/starred which makes it quite uncomfortable to try to diff results of similar views.

FreshRSS's web views have separate binary toggles for show-read, show-unread, show-starred and sort-asc/dec which all operate independently and which enable fully exploring the state of a collection of items in confusing cases. In addition to being useful on a day to day basis, this ability to modify and compare feed views flexibly has helped me when I had misaligned expectations or was working around a bug.
